### ISSUE TITLE
Volumes are not deleted if app was renamed before

### ISSUE THREAD

**Issue by Nikdro** [caprover/caprover#846]
**What is the problem?**
If you create an app from one of the templates and rename it afterwards, the volumes are not getting deleted after deleting the app from caprover.

Steps to reproduce the problem:
1. Create an app from the templates.
2. Change the name of the app.
3. Delete the app.
-> A window with the information, that volumes are not getting deleted appears.

Not sure if that is a general problem or related to my system. Maybe someone can try to reproduce and report here?

**Comment by andrew-ma** [comment]
yes same for me. I just use portainer to delete volumes, and it shows unused in portainer so it is easy
